### PR TITLE
optionsAdmin

### PR DESCRIPTION
**Pull request by lxc150896**
task: https://dev.framgia.com/issues/217303

### CODE REVIEW

File: app/Http/Controllers/SettingController.php [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        $condition = [
+            'id' => $id,
+        ];
+        $data = $this->option->getData($condition)->first();
+        
+        return $data->value;
+    }
+
+    public function postAddApp(Request $request)
+    {
```
Comment (framgiaciv3) [inline app/Http/Controllers/SettingController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/SettingController.php</br>**Line**: 89</br>Blank line found at start of control structure

File: app/Http/Controllers/SettingController.php [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        try {
+            if ($request->idTextApp == null) {
```
Comment (framgiaciv3) [inline app/Http/Controllers/SettingController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingAfterOpen</br>**Severity**: error</br>**File**: app/Http/Controllers/SettingController.php</br>**Line**: 91</br>Blank line found at start of control structure

File: resources/assets/admin/js/setting.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
+$.ajaxSetup({
+    headers: {
+        'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+    }
+});
+
+$('#text-footer').on('click', function() {
+    var value = $('#message').val();
+    if (($('#message').val()) == '') {
+        $( "#message" ).addClass("border-danger");
```
Comment (BunnyPi04) [inline resources/assets/admin/js/setting.js]: nháy đơn
khoảng trắng

File: resources/views/admin/setting/option.blade.php [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+                        @if (isset($textApps[$key]))
+                        @include ('admin.setting.edit_app')
+                        @endif
+                        @endforeach
+                        @endif
+                    </div>
+                </div>
+            </div>
+            <div class="col-xl-4">
+                <div class="m-portlet m-portlet--full-height m-portlet--fit ">
```
Comment (danhloimta) [inline resources/views/admin/setting/option.blade.php]: khoảng trắng thừa.

File: resources/views/admin/setting/option.blade.php [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+                                    @include ('admin.setting.option_edit')
+                                @endforeach
+                            </ul>
+                            @endif
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="col-xl-4">
+                <div class="m-portlet m-portlet--full-height m-portlet--fit ">
```
Comment (danhloimta) [inline resources/views/admin/setting/option.blade.php]: thừa khoảng trắng

File: resources/views/admin/setting/option.blade.php [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+                                </h3>
+                            </div>
+                        </div>
+                        <div class="m-portlet__head-tools">
+                            <a class="btn btn-accent m-btn m-btn--custom m-btn--icon m-btn--pill m-btn--air">
+                                <span>
+                                    <i class="la la-plus"></i>
+                                    <span data-toggle="modal" data-target="#myModalApp">{{ trans('admin.option.addApp') }}</span>
+
+                                </span>
```
Comment (danhloimta) [inline resources/views/admin/setting/option.blade.php:96]: thừa khoảng trắng

File: resources/views/admin/setting/option.blade.php [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+                        <div class="news">
+                            @if (isset($banners) && count($banners) > 0)
+                            @foreach ($banners as $key => $banner)
+                            <div class="article">
+                                {!! Form::open(['method' => 'POST', 'id' => 'uploatImg' . $banner->id, 'files' => true]) !!}
+                                {!! Form::hidden('banner', $banner->id) !!}
+                                {!! Form::file('img', ['values' => $banner->id, 'class' => 'form-control hidden img-file', 'id' => 'banner' . $banner->id, 'required', 'onchange' => 'changeImg(this)', 'accept' => 'image/png, image/jpeg, image/jpg']) !!}
+                                <img values="{{ $banner->id }}" id="img-banners{{ $banner->id }}" class="thumb avatar{{ $key }}" src="{{ asset(config('view.image_paths.slide') . $banner->value) }}">
+                                {!! Form::close() !!}
+                                <div class="title text-banner">{{ trans('admin.option.banner') }} {{ $key +1 }}</div>
```
Comment (danhloimta) [inline resources/views/admin/setting/option.blade.php]: convention

**Pull request merged**